docs/src/threadpool.rst

```diff
@@ -17,6 +17,8 @@ is 1024).
 .. versionchanged:: 1.45.0 threads now have an 8 MB stack instead of the
    (sometimes too low) platform default.
 
+.. versionchanged:: REPLACEME threads now have a default name of libuv-worker.
```

Comment:
How are those next-tags handled here? Should I hardcode the next one?

Reply:
Yes. `1.50.0` if I'm not mistaken